### PR TITLE
BaSP-TG-39: Employees List

### PR DESCRIPTION
**Pull request by gc796**
- Used list component
- Used button component
- Set list and form in different screens
- Function inside useEfect is not asynchronous

### CODE REVIEW

**Review by Evelyn-Heredia — APPROVED** [review]
PR approved!

**Review by Julian25 — APPROVED** [review]

File: src/Components/Employees/List/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  ) : (
+    <section className={styles.container}>
+      <h2>Employees</h2>
+      <List
+        fullList={employeesList}
+        data={formatListData(employeesList)}
+        headers={headers}
+        resource={resource}
+        deleteItem={deleteEmployee}
+        method={method}
```
Comment (BaltazarCortina1) [inline src/Components/Employees/List/index.js:80]: List no recibe ninguna prop method

File: src/Components/Employees/List/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  useEffect(() => {
+    getEmployees();
+  }, []);
+
+  const getEmployees = async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_URL}${resource}`);
+      const jsonResponse = await response.json();
+      setEmployeesList(jsonResponse.data);
```
Comment (BaltazarCortina1) [inline src/Components/Employees/List/index.js:22]: Sigue faltando validar la respuesta

File: src/Components/Employees/List/index.js [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import List from '../../Shared/List/List';
+import Button from '../../Shared/Button/Button';
+import Loading from '../../Shared/Loading/Loading';
+import styles from './employees.module.css';
+
+const Employees = () => {
+  const [employeesList, setEmployeesList] = useState([]);
+  const [method, setMethod] = useState('POST');
```
Comment (BaltazarCortina1) [inline src/Components/Employees/List/index.js:10]: Esto no es necesario

**Pull request merged**